main.py

```diff
@@ -1,10 +1,41 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player, player_info in players.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_info["race"].get("name"),
+            description=player_info["race"].get("description")
+        )
+
+        for skill in player_info["race"].get("skills"):
+            Skill.objects.get_or_create(
+                name=skill.get("name"),
+                bonus=skill.get("bonus"),
+                race=race
+            )
+
+        guild = player_info.get("guild")
```

Comment:
The variable 'guild' is being overwritten in the if block. Consider using a different variable name for the result of 'get_or_create' to avoid confusion.